src/Utils/Proxy.js

```diff
@@ -0,0 +1,5 @@
+//  SERVER ROUTES
+
+export const  SERVER = "https://nodejs-teamwork.herokuapp.com/"
```

Comment:
the server url is enough don't add the server auth url, when you are doing auth just pass in the other parameters

Reply:
Ok sir. I have made the changes